src/main/java/core/basesyntax/model/InvalidOperationCodeException.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.model;
+
+public class InvalidOperationCodeException extends RuntimeException {

```

Comment:
Don't begin class implementation with an empty line. Please remove the redundant empty line at the beginning of the class definition.